### ISSUE TITLE
helper module is not working

### ISSUE THREAD

**Issue by parasjain320** [dr-costas/mad-twinnet#11]
%matplotlib inline 
import helper

images, labels = next(iter(trainloader))

img = images[0].view(1,784)
#turn off gradient to sppped  up this part
with torch.no_grad():
    logits = model.forward(img)
    
ps = F.softmax(logits, dim= 1)
helper.view_classify(img.view(1,28,28), ps)

**Comment by dr-costas** [comment]
Hi

I cannot understand what is the problem. 

To begin with, there is not `view_classify` at the helper**s** (and not helper) module. Secondly, the system MaDTwinNet is not a classification system and, most probably, you are doing something wrong if you are using the softmax after the output of the system. 

Can you please describe what is exactly that you want to do and what is the exact problem?

**Comment by dr-costas** [comment]
@parasjain320 
Can you please confirm or disprove if your issue still holds?

**Comment by dr-costas** [comment]
@parasjain320 
I close the comment sine: 

1. The comment is not about something in the code of the repository
2. The problem is not properly explained 
3. There is not any activity towards the resolution of the presented problem

If there is something else about this repository, please create another issue.
